### ISSUE TITLE
Potential string interpolation issues in resources/tests/setup_rootfs.sh

### ISSUE THREAD

**Issue by wimax-grapl** [firecracker-microvm/firecracker#2932]
# Describe the bug

setup_rootfs.sh has the following:
```
cat <<EOF > "/etc/systemd/system/serial-getty@ttyS0.service.d/autologin.conf"
[Service]
ExecStart=
ExecStart=-/sbin/agetty --autologin root -o '-p -- \\u' --keep-baud 115200,38400,9600 %I $TERM
EOF
```

-----

Two problems with this:

1. Incorrectly escaped \\u
The double-\ gets interpreted by the Bash script and becomes a single \ within the written file.
A [cursory search](https://github.com/search?q=%22-%2Fsbin%2Fagetty+--autologin+root+-o%22&type=code) suggests this is incorrect and the resulting file _should_ have a double-\

I first discovered this because upon starting up the rootfs generated from setup_rootfs.sh you get
```
[    2.007025] systemd[1]: /etc/systemd/system/serial-getty@ttyS0.service.d/autologin.conf:3: Ignoring unknown escape sequences: "-p -- \u"
```

-----

2. Incorrectly escaped $TERM
 the TERM gets interpolated by the script running the cat <<EOF, not at systemd time. 

Running `less` on this autologin.conf in firecracker results in the following:
`%I unknown`

## Checks

- [x] Have you searched the Firecracker Issues database for similar problems?
- [x] Have you read the existing relevant Firecracker documentation?
- [x] Are you certain the bug being reported is a Firecracker issue?

**Comment by dianpopa** [comment]
Mistakenly closed the wrong issue

**Comment by pb8o** [comment]
Fixed in d8e52c8561cdf9aac9b8d3ada5aa66afc096b970
